app/assets/javascripts/availability.js.coffee

```diff
@@ -0,0 +1,29 @@
+jQuery ->
+  window.availability_updater = new AvailabilityUpdater
+class AvailabilityUpdater
+  constructor: ->
+    this.request_availability()
+  availability_url: "http://bibdata.princeton.edu/availability"
+  request_availability: ->
+    ids = this.record_ids().toArray()
+    params = $.param({ids: ids})
+    url = "#{@availability_url}?#{params}"
+    $.getJSON(url, this.process_records)
+  process_records: (records) =>
+    for record_id, availability_info of records
+      this.apply_record(record_id, availability_info)
+  apply_record: (record_id, availability_info) ->
+    return if availability_info.more_holdings == true
```

Comment:
So there's two kinds of more holdings: There's the view determined it has more holdings, so it doesn't display anything, and then there's the view displays holdings, but the API shows "more_holdings: true". Do both cases need that message?